products/bash/release-ace-integration-server.sh

```diff
@@ -107,13 +110,24 @@ echo -e "\n---------------------------------------------------------------------
 # -------------------------------------- FIND TOTAL ACE REPLICAS DEPLOYED -----------------------------------------------
 
 numberOfReplicas=$(oc get integrationservers $is_release_name -n $namespace -o json | ./jq -r '.spec.replicas')
-echo "INFO: Number of Replicas for $is_release_name is $numberOfReplicas"
-echo -e "\nINFO: Total number of ACE integration server $is_release_name related pods after deployment should be $numberOfReplicas"
+echo "INFO: Number of Replicas for '$is_release_name' is $numberOfReplicas"
+echo -e "\nINFO: Total number of ACE integration server '$is_release_name' related pods after deployment should be $numberOfReplicas"
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
-# -------------------------------------- CHECK FOR NEW IMAGE DEPLOYMENT STATUS ------------------------------------------
+# ------------------------------------------------ FIND IMAGE TAG --------------------------------------------------
+
+imageTag=${is_image_name##*:}
+
+echo "INFO: Image tag found for '$is_release_name' is '$imageTag'"
+
+if [[ -z "$imageTag" ]]; then
```

Comment:
I think it would be best to have this check at the top of the script, validate the input parameters before doing anything

Reply:
Done